tools/create_virtual_hardware.sh

```diff
@@ -63,7 +63,7 @@ function ensure_softnpu_zone {
             --omicron-zone \
             --ports sc0_0,tfportrear0_0 \
             --ports sc0_1,tfportqsfp0_0 \
-            --sidecar-lite-commit 45ed98fea5824feb4d42f45bbf218e597dc9fc58 \
+            --sidecar-lite-commit 6007a1b0ffe57ee738222867c343bde7e47bbf60 \
```

Comment:
i don't see this commit in the sidecar-lite commit history.

Reply:
Ack, that was the pre-merge branch. Should be remedied now.